src/freelunch/optimisers.py

```diff
@@ -75,33 +83,67 @@ class SADE(continuous_space_optimiser):
         'F_sig':0.2,
         'Cr_u':0.5,
         'Cr_sig':0.2,
-        'Lp':10
+        'Lp':10,
+        'XOver':'binary'
     }
 
+    # Exportable dictionary
+    DE_methods = [x() for x in darwin.adaptable_search_operation.__subclasses__()]
+
+
+    # API for probability update
+    def update_strategy_ps(self,strats):
```

Comment:
My plan was to have this be in darwin / tech. Several other MHs use adaptable parameters and I thought it would be good to have this work plug n play?

Reply:
This is why I was trying to build a random/adaptive mutation strategy inheriting from adaptable_search_operation but the way on_win works makes it tricky...